libs/providers/flagsmith-client/package.json

```diff
@@ -12,6 +12,6 @@
   },
   "peerDependencies": {
     "@openfeature/web-sdk": "^1.0.0",
-    "flagsmith": "4.0.0"
+    "flagsmith": "^4.0.3"
```

Comment:
@dabeeeenster @matthewelwell this seemed like a very strict peer. I think you might wan to use a major version matcher.

Let me know if you object.